src/main/java/nl/smallproject/www/techiteasy/repositories/RemoteControllerRepository.java

```diff
@@ -0,0 +1,10 @@
+package nl.smallproject.www.techiteasy.repositories;
+
+import nl.smallproject.www.techiteasy.models.RemoteController;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
```

Comment:
Deze annotatie is hier niet nodig, omdat je al extend van JpaRepository waarmee je deze annotatie ook overerft